www/artists/dloran/ContentFlow/contentflow.js

```diff
@@ -1,2 +1 @@
-/* ContentFlow, version 1.0.2  | (c) 2007 - 2010 Sebastian Kutsch | <http://www.jacksasylum.eu/ContentFlow/> | ContentFlow is distributed under the terms of the MIT license. | (see http://www.jacksasylum.eu/ContentFlow/LICENSE) */
```

Comment:
I'm pretty sure this comment at the top is important. Unless you actually replaced the script with something you wrote yourself, we need to include the credit to the actual author.